src/oop/object_independence/medium/q4/QuestionText.md

```diff
@@ -0,0 +1,33 @@
+# oop-object_independence-medium-q4
+
+## Question Info
+
+- Question type: Java
+
+### Question Text
```

Comment:
From the way the description is worder, it seems like the user can assume that the string array has at least one item in it. Can you please mention this?

Reply:
I don't think the `playNext` and `playPrev` methods will work otherwise.